Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs

```diff
@@ -518,6 +522,18 @@ private void AppxBuildGUI()
                 EditorUserBuildSettings.wsaArchitecture = newBuildArchitectureString;
             }
 
+            // The 'Gaze Input' capability is newly added for the Windows SDK 18362, but existing versions
```

Comment:
Nit: The capability was added earlier for preview APIs for desktop eye trackers. Support on HoloLens was added in 18362.

```suggestion
            // The 'Gaze Input' capability is newly added for the Windows SDK 17134, but existing versions
```